app/src/main/java/com/zeapo/pwdstore/git/GitConfigActivity.kt

```diff
@@ -66,4 +54,51 @@ class GitConfigActivity : BaseGitActivity() {
             }
         }
     }
+
+    /**
+     * Sets up the UI components of the tools section.
+     */
+    private fun setupTools() {
+        val repoDir = PasswordRepository.getRepositoryDirectory()
+        val repo = PasswordRepository.getRepository(repoDir)
```

Comment:
Blame is on us, but this can actually be "simplified" to:
```suggestion
        val repo = PasswordRepository.getRepository(null)
```